documentation/infra/environment-variables-and-secrets.md

```diff
@@ -0,0 +1,58 @@
+# Environment variables and secrets
```

Comment:
https://github.com/navapbc/template-infra/blob/main/docs/infra/environment-variables-and-secrets.md